config.toml.example

```diff
@@ -171,37 +176,37 @@ changelog-seen = 2
 # first compiler.
 #
 # Defaults to host platform
-#build = "x86_64-unknown-linux-gnu"
+#build = <host platform>
 
 # Which triples to produce a compiler toolchain for. Each of these triples will
 # be bootstrapped from the build triple themselves.
 #
 # Defaults to just the build triple
-#host = ["x86_64-unknown-linux-gnu"]
+#host = [build]
 
 # Which triples to build libraries (core/alloc/std/test/proc_macro) for. Each of
 # these triples will be bootstrapped from the build triple themselves.
 #
 # Defaults to `host`. If you set this explicitly, you likely want to add all
 # host triples to this list as well in order for those host toolchains to be
 # able to compile programs for their native target.
-#target = ["x86_64-unknown-linux-gnu"]
+#target = host (array)
```

Comment:
"configured host" or something like this, otherwise it's unclear whether this is defaulting to the host platform (i.e., build = ) or the host array above.

I'm not actually convinced that the diffs to these over stating the linux build triple is actually helpful, it feels like it might just confuse people more. We could instead add a comment that this is just an example configuration, but will differ based on the platform being compiled on.

Reply:
This doesn't seem done, unless I'm missing something.

Reply:
I seem to have lost these commits somewhere; I'm just going to rewrite the changes.